docs/01-overview/main-areas/service-mesh/smsh-04-istio-sidecars-in-kyma.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Istio sidecars in Kyma and why you want them
+---
+
+By default, Istio installed as part of Kyma [is configured](./smsh-02-default-istio-setup-in-kyma.md) with automatic Istio proxy sidecar injection enabled. This means that all Pods of your workloads (such as deployments and StatefulSets; except any workloads in the `istio-system` or `kube-system` Namespaces) get their own sidecar proxy container running next to your application. With an Istio sidecar, the resource becomes part of Istio service mesh, which brings the following benefits that would be complex to manage otherwise.
+
+
+
+## Secure communication
+
+In Kyma's [default Istio configuration](./smsh-02-default-istio-setup-in-kyma.md), [peer authentication](https://istio.io/latest/docs/concepts/security/#peer-authentication) is set to cluster-wide `STRICT` mode. This ensures that your workload only accepts [mutual TLS traffic](https://www.cloudflare.com/learning/access-management/what-is-mutual-tls/) where both, client and server certificates, are validated to have all traffic encrypted. This provides each service with a strong identity, with reliable key and certificate management system.
+
+Another security benefit of having a sidecar proxy is that you can perform request authentication for your service. Istio enables [request authentication](https://istio.io/latest/docs/reference/config/security/request_authentication/) with JSON Web Token (JWT) validation using a custom authentication provider. Learn how to [secure your service using JWT](../../../03-tutorials/00-api-exposure/apix-05-expose-and-secure-workload-jwt.md).
```

Comment:
We might want to explain that at the first occurrence, not the second one :D 

```suggestion
Another security benefit of having a sidecar proxy is that you can perform [request authentication](https://istio.io/latest/docs/reference/config/security/request_authentication/) for your service. Istio enables request authentication with JSON Web Token (JWT) validation using a custom authentication provider. Learn how to [secure your workload using JWT](../../../03-tutorials/00-api-exposure/apix-05-expose-and-secure-workload-jwt.md).
```
Title of the link target says "workload", so I'd use that here, too.